src/server.js

```diff
@@ -18,6 +18,11 @@ var authFunction = localAuth;
 module.exports = function() {
 	manager = new ClientManager();
 
+	if (!fs.existsSync("client/js/bundle.js")) {
```

Comment:
Yeah, let's look into this in another one. I guess you could do it by the first line of the bundle being a version number or git hash  or something.